scripts/log_analysis.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+
+# read in log csv
+log_file = "./logs-insights.csv"  # path to logs as .csv
```

Comment:
Can we document how to generate this file?

Reply:
Absolutly, will include this. They are downloaded directly from CloudWatch, so can provied a step by step

Reply:
Let me know if you want to pair on that as you don't have access @Connor-OS 👍